tests/phpunit/tests/rest-api/wpApplicationPasswords/createNewApplicationPassword.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Unit tests covering WP_Application_Passwords::create_new_application_password functionality.
+ *
+ * @package    WordPress
+ * @subpackage REST API
+ */
+
+/**
+ * @covers WP_Application_Passwords::create_new_application_password
+ *
+ * @group  restapi
+ * @group  app_password
+ */
+class Test_WPApplicationPasswords_CreateNewApplicationPassword extends WP_UnitTestCase {
```

Comment:
```suggestion
class Test_WP_Application_Passwords extends WP_UnitTestCase {
```

I think we can put both of these tests into a single class covering `WP_Application_Passwords`.

Reply:
Per the PHPUnit Testing Standard, each public method is supposed to have its own test file: [here](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-methods) and here](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes). This is an initiative I'll be tackling in 2021, ie to update all of the tests to comply.

Reply:
That said, if you'd prefer:

- we can consolidate the tests now
- then in a different ticket rename and split as part of a separate test standards compliance initiative.

Reply:
Hm, my understanding from @SergeyBiryukov's [comment](https://wordpress.slack.com/archives/C02RQBWTW/p1610651982043100?thread_ts=1610648341.040700&cid=C02RQBWTW) was that we'd do the opposite.

Reply:
Let's for now consolidate. Then in a later initiative, we can come together to figure out the testing standard and either change the tests or update the standard.

>As a rule, a single test class should not contain tests for more than one function/method.

Reply:
Sounds good to me.

Reply:
Consolidated into one test class in commit https://github.com/WordPress/wordpress-develop/pull/898/commits/190e1d73e560002f69019ad9463810ad9055d1cf and renames the file to match REST API test convention https://github.com/WordPress/wordpress-develop/pull/898/commits/3aaedf3cadc87f8e74e3546840741b81f1e767fa.